src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import core.basesyntax.interfaces.AreaCalculator;
+import core.basesyntax.interfaces.Describer;
+
+public abstract class Figure implements Describer, AreaCalculator {
+    public String type = "circle";
+    public String color = "WHITE";
```

Comment:
The fields 'type' and 'color' should be private to follow the encapsulation principle. Additionally, if these fields are common to all figures and are not supposed to change, they should be declared as constants (final). If they are supposed to change, they should be instance variables without initialization here, and you should provide a constructor to set their values.